01-html/05-aol.html

```diff
@@ -2,4 +2,130 @@
 I picked a signup form for something that I (hope) you don't already have an account for.
 
 Forms are important!
--->
\ No newline at end of file
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AOL | Sign Up</title>
+  </head>
+<body>
+  <header>
```

Comment:
This might more appropriately be set to be `nav` or just a `div` - `header` implies that this is the header for the page. A search engine, for example, might look for a `header` tag to find the page title inside.